test/libsolidity/syntaxTests/pragma/invalid_range_conjunction_1.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity 0.4.1 - 0.6.2 0.8.17;
```

Comment:
Please give these tests more detailed names. Avoid just adding numbers - we have a lot of that and it make it harder to see which cases we have covered. It's ok to leave existing ones as is but new ones should not use that convention.

I'd suggest naming it like this:
- `invalid_range_conjunction_1.sol` - > `invalid_range_conjunction_range_first.sol`
- `invalid_range_conjunction_2.sol` - > `invalid_range_conjunction_range_second.sol`
- `invalid_range_conjunction_3.sol` - > `invalid_range_conjunction_two_ranges.sol`
- `valid_range_disjunction_1.sol` -> `valid_range_disjunction_range_second.sol`
- `valid_range_disjunction_2.sol` -> `valid_range_disjunction_two_ranges.sol`

Reply:
Right. I will remember that for future PRs.